rspace/src/main/scala/coop/rchain/rspace/RSpace.scala

```diff
@@ -77,6 +77,7 @@ class RSpace[F[_], C, P, A, R, K] private[rspace] (
                 patterns,
                 continuation,
                 persist,
+                Seq.empty,
```

Comment:
Why have you suddenly switched to using `Seq` instead of `Set`?

Reply:
serialization. I'm working on a version that works with sorted sets.